src/Nancy/ModelBinding/DefaultBinder.cs

```diff
@@ -93,20 +92,29 @@ public object Bind(NancyContext context, Type modelType, object instance, Bindin
             var bindingContext =
                 this.CreateBindingContext(context, modelType, instance, configuration, blackList, genericType);
 
-            var bodyDeserializedModel = this.DeserializeRequestBody(bindingContext);
+            var bindingExceptions = new List<PropertyBindingException>();

```

Comment:
That is a good question. I think I wanted to add an exception to the list at some point, but decided to just throw right away instead. I'll move it back :smile: